__tests__/tests/components/EntityInfo.js

```diff
@@ -20,13 +20,6 @@ describe('<EntityInfo />', () => {
     expect(wrapper.find({ title: 'Location' }).prop('subtitle')).toEqual(
       organization.location
     );
-    wrapper.find({ title: 'Location' }).simulate('press');
-    expect(Communications.web).toBeCalledWith(
-      `https://www.google.com/maps/place/${organization.location.replace(
-        / /g,
-        '+'
-      )}`
-    );
```

Comment:
Try to test your new press action. Mock `navigation.navigate` and check whether it is called with right arguments.